wheatley/parsing.py

```diff
@@ -202,7 +208,7 @@ def json_to_call(name):
         except KeyError as e:
             raise_error('stage', "'stage' is not defined", e)
         except ValueError as e:
-            raise_error('stage', f"'{json[stage]}' is not a valid integer", e)
+            raise_error('stage', f"'{json['stage']}' is not a valid integer", e)
```

Comment:
:+1: Useful spot

Reply:
Yeah the type checker pulled it up immediately - gotta love static typing...